client/extensions/tga-author-profile-fields/src/components/vocab/editor.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+import {ILiveResourcesProps, IRestApiResponse, IVocabulary} from 'superdesk-api';
+import {superdesk} from '../../superdesk';
+import {IVocabularyFieldProps} from './interfaces';
+
+import {Select, Option} from "superdesk-ui-framework/react";
+
+
+export class VocabularyField extends React.PureComponent<IVocabularyFieldProps> {
+    render() {
+        const {WithLiveResources} = superdesk.components;
+        const resources: ILiveResourcesProps['resources'] = [
```

Comment:
It's better to return `null` from the render method if `this.props.config` is null.
Otherwise, using optional chaining would result in `WithLiveResources` getting called with the following:
`resources={[{resource: 'vocabularies', ids: [undefined]}]}`